pkg/apis/v2v/v1alpha1/virtualmachineimport_types.go

```diff
@@ -22,6 +22,9 @@ type VirtualMachineImportSpec struct {
 	// +optional
 	TargetVMName *string `json:"targetVmName,omitempty"`
 
+	// +optional
+	VMDescription *string `json:"vmDescription,omitempty"`
```

Comment:
should we consider renaming the fields so they do not include `vm` inside? 

Especially in case if we end up importing templates (not sure about vmware part) as well?

Reply:
I can suggest an alternate method for it: instead of including it as part of the API, we can define annotation that will contain a list of key-values map as its value and its purpose will be to propagate it to the target vm, without the need to specify the name of the fields explicitly. 

in this case, we can agree that if there is an annotation named `vmimport.v2v.kubevirt.io/propagate-annotations` - its values will be  propagated to target VM

An example of it in a vm-import-cr could look like:
```
apiVersion: v2v.kubevirt.io/v1alpha1
kind: VirtualMachineImport
metadata:
  name: example-virtualmachineimport
  namespace: example-ns
  annotations:
    vmimport.v2v.kubevirt.io/propagate-annotations:
      "description": "the-resource-description",
      "another-annotation": "...",
...
```

Reply:
That was something I wanted to do at the beginning but @suomiy suggested to have it in spec. We could have common logic to pass annotations and labels and in this way we could fix https://github.com/kubevirt/vm-import-operator/issues/138 as well. @suomiy what do you think?

Reply:
I agree something like this would be very useful - also for the labels.

anyway I am not sure the implementation you are proposing is feasible.

The annotations are expected to be mapping of string to string. Not sure if you can change it in your definition, but all other projects/resources expected this to be a string.

Maybe we could create a new field for this?

Reply:
We could use json string instead. It should not be that difficult to create it in the UI. @masayag what do you think?

Reply:
It can simply be multi-line value to be parsed into a map, my example should be changed to contain the multi-line notation by adding | after `    vmimport.v2v.kubevirt.io/propagate-annotations:
`

Reply:
Yes, not difficult for the UI. Just harder for people trying to use YAML

Reply:
I do not think it would be used from command line very heavily.

Reply:
Also, the user will be able to edit the VM resource after it was created instead of relying on the vm-import process to do so.